src/pages/dashboard/DashboardPage.tsx

```diff
@@ -70,21 +75,53 @@ const DashboardPage = () => {
     gtfs_route_hour: item.gtfs_route_hour,
   }))
 
+  const datePickerErrorMessage = React.useMemo(() => {
```

Comment:
useMemo isn't needed here (primitive value, and easy computation)

Reply:
fixed